URL_Builders/Arxiv_URLBuilder.ipynb

```diff
@@ -0,0 +1,659 @@
+{
```

Comment:
I don't think that adding .ipynb files to the main project branch adds to the project. It's better to keep the main branch cleaner. I suggest to remove this one and the other .ipynb files